### DISCUSSION TITLE
Achieve bash transparency T_T (edit)

### DISCUSSION THREAD

**Discussion by lash0000** [end-4/dots-hyprland#144] [Q&A]
![image](https://github.com/end-4/dots-hyprland/assets/38674978/e603b3e4-fee9-4987-839a-e91f2755753c)

Just got finally to use this kind of awesome hyprdot I rate it 10/10 massively.

![image](https://github.com/end-4/dots-hyprland/assets/38674978/d5ece7d3-797b-4a6a-a6cf-820f6ba1a69d)

Done with my first question which is to my locate my PATH directory sorry i'm stupid

So how can I add some transparency in my bash terminal how can I achieve it?

**Comment by lash0000** [comment]
So yeah, got my answers to it so I found it in `.config/foot` with the [colors] inside alpha=0

Sorry for disturbing T_T

**Reply by end-4** [reply]
You might also wanna change `~/.config/ags/scripts/templates/foot/foot.ini`
This is the template file which will be applied when you pick another wallpaper and color generation happens

**Reply by lash0000** [reply]
All good now sir thanks for awesome dotfile (hopefully not having a conflicts when there's new version for hyprland itself) ^^

**Reply by LunnaBoo** [reply]
hey, i got the same question, but for me changing the 'alpha' value in `/.config/foot/foot.ini` isn't really doing anything :(( 
i also tried to find the `~/.config/ags/scripts/templates/foot/foot.ini` but the foot folder under templates does not exist 
please help

**Reply by end-4** [reply]
- make sure you restarted foot after changing the config
- if the template file isn't there, it's likely that you missed a step while installing. you can either 
    - run it again, skipping the parts you've already run successfully and say yes when it asks to copy the (ags) files
    - or just manually copy the files from the repo to your system

**Reply by LunnaBoo** [reply]
![image](https://github.com/user-attachments/assets/6696e143-ce4c-4f1b-aca0-e8b83125f4a8)
there's just nothing there i can't copy from the repo cause it's not there in ags

**Reply by end-4** [reply]
OH
i removed foot config from templates long ago because now we use escape sequences for terminal colors. now only editing ~/.config/foot/foot.ini should suffice
my former tip still applies, idk what you should do if that doesn't work

**Reply by Dillpickleschmidt** [reply]
Go to ~/.config/ags/scripts/color_generation/applycolor.sh and change the term_alpha to something lower like 90. Then load a wallpaper to trigger the script. This should work for most terminals (not just foot).

You're welcome ;)

*you might have to wiggle the terminal around in hyprland. It doesn't seem to apply for me when it first opens.

see this for tmux config to match: [#1265](https://github.com/end-4/dots-hyprland/discussions/1265#discussioncomment-13265924)
